src/ProgressOnderwijsUtils/ProgressOnderwijsUtils.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\NugetPackagesCommon.props" />
   <PropertyGroup Label="Configuration">
-    <Version>60.0.0</Version>
-    <PackageReleaseNotes>Improve nullabilty annotations</PackageReleaseNotes>
+    <Version>61.0.0</Version>
+    <PackageReleaseNotes>RandomHelper insecure to private method</PackageReleaseNotes>

```

Comment:
more like ' RandomHelper removed from public api'

Reply:
Dat is inderdaad beter